third-party/gmp/Makefile

```diff
@@ -28,7 +28,7 @@ CHPL_GMP_CFG_OPTIONS += --enable-static --disable-shared
 endif
 
 CHPL_GMP_ABI_ARG =
-ifeq (linux32, $(CHPL_MAKE_TARGET_PLATFORM))
+ifeq ($(CHPL_MAKE_TARGET_PLATFORM), $(filter $(CHPL_MAKE_TARGET_PLATFORM),linux32 cygwin32))
```

Comment:
Oh, I guess in that case someone would set CHPL_BUILD_X86_64_AS_32 in the environment and the platform value would be linux64_32. So nevermind.